### DISCUSSION TITLE
HTML的發明人：提摩西·約翰·柏內茲-李爵士 (Sir Timothy John Berners-Lee)

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#18] [General]
要有怎麼樣的機緣才可以創造出影響後世的HTML文件格式?

來看看 [HTML的發明人：提摩西·約翰·柏內茲-李爵士 (Sir Timothy John Berners-Lee) 的故事吧!](https://zh.wikipedia.org/wiki/%E8%92%82%E5%A7%86%C2%B7%E4%BC%AF%E7%BA%B3%E6%96%AF-%E6%9D%8E)

![image](https://user-images.githubusercontent.com/1768891/134809725-e1d6a4e6-ea38-4020-9bde-39a8e77d215e.png)

**Comment by Jie2222** [comment]
感覺它創造了另一個宇宙的空間，而且居然還不申請專利，目的就是為了讓資訊網普及，太偉大了...

**Comment by enoch-lightning** [comment]
發明HTML的人居然還在世，且才剛步入老年。由此來看，很多電腦的相關知識，應該都是從20世紀開始發展的。雖電腦在科學歷史的進程上，很晚出現，卻為世界帶來無比的方便，提昇文書處理、公文傳送與科學計算的效率。

**Comment by xCk27x** [comment]
柏內茲・李 :「我只要把超文字系統和傳輸控制協定、域名系統結合在一起，然後─搭啦！─就有了全球資訊網」
全球資訊網並非全部從頭開發，而是將各項技術結合在一起進行廣義化的運用後產生出來的，而這也凸顯了廣泛吸收四面八方的資訊有多麼重要。

**Comment by SamTU0222** [comment]
他說到，  「網路中立性是一種網路人權，如果公司、政府干涉或監視網際網路流量，基本人權會受到威脅」，其實網路的使用上也與人權息息相關。
